Consul.Test/OperatorTest.cs

```diff
@@ -100,9 +102,20 @@ public async Task Segment_List()
         [EnterpriseOnlyFact]
         public async Task Operator_CreateArea()
         {
-            var check = new AreaRequest { PeerDatacenter = "dc2", UseTLS = false, RetryJoin = null };
+            var peerDataCenter = KVTest.GenerateTestKeyName();
+            var check = new AreaRequest { PeerDatacenter = peerDataCenter, UseTLS = false, RetryJoin = null };
+
             var response = await _client.Operator.CreateArea(check);
             Assert.NotNull(response.Response);
         }
+        [EnterpriseOnlyFact]
+        public async Task Operator_AreaList()
+        {
+            var peerDataCenter = KVTest.GenerateTestKeyName();
+            await _client.Operator.CreateArea(new AreaRequest { PeerDatacenter = peerDataCenter, UseTLS = false, RetryJoin = null });
+
+            var req = await _client.Operator.AreaList();
+            Assert.NotEmpty(req.Response);
```

Comment:
Just one more thing, can we test if the fields in the response are not null? 
So far we know that we can create and list areas, but it would be great if we assert that the values returned by the list operation correspond to the values used in the create operation.

Reply:
✅ done

Reply:
Can you also try filling the `RetryJoin` field and then validating the content of it when returned?